deployments/xpumanager_sidecar/kustom/kustom_xpumanager.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: intel-xpumanager
+  name: intel-xpumanager
+spec:
+  template:
+    spec:
+      volumes:
+        - name: features-d
+          hostPath:
+            path: "/etc/kubernetes/node-feature-discovery/features.d/"
+      containers:
+      - name: xelink-sidecar
+        image: intel/intel-xpumanager-sidecar:devel
+        imagePullPolicy: Always
+        args:
+          - -v=2
+        volumeMounts:
+        - name: features-d
+          mountPath: "/etc/kubernetes/node-feature-discovery/features.d/"
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          readOnlyRootFilesystem: true
+          runAsUser: 0
+      - name: xpumd
+        resources:
+          limits:
+            $patch: replace
+            gpu.intel.com/i915_monitoring: 1
```

Comment:
As you're dropping extra resources, maybe drop also extra volume mounts?

Reply:
I'm dropping resources because XPU-Manager by default depends on "smarter-device-manager": https://github.com/intel/xpumanager/blob/master/deployment/kubernetes/daemonset-intel-xpum.yaml#L47 . I don't think the volumes hurt where the extra resource will prevent deployment.

Reply:
Hmm.. But maybe the "patch installation" should use a different yaml file so it doesn't remove the smarter-device-manager if the user has it enabled.

Reply:
I decided to add a NOTE.

Reply:
Ok, I'm fine with that.